tests/ext2/crypt/Makefile

```diff
@@ -54,13 +54,10 @@ MKEXT2_OPTS += --force
 MKEXT2_OPTS += --encrypt=$(KEYFILE)
 MKEXT2_OPTS += -p=passphrase12345
 
-OUTFILE=$(SUBOBJDIR)/outfile
-
 $(IMAGE):
 	mkdir -p $(SUBOBJDIR)
 	sudo $(MYST) mkext2 $(MKEXT2_OPTS) ext2dir $(IMAGE)
-	$(MYST) fssig $(IMAGE) > $(OUTFILE)
-	@ grep root_hash= $(OUTFILE) | cut -d= -f 2 > $(ROOTHASH)
+	$(MYST) fssig --roothash $(IMAGE) > $(ROOTHASH)
```

Comment:
I thought we don't need the roothash in a separate file anymore.

Reply:
We still need it when the app is explicitly mounting the ext2 file system (as in this case). In the rootfs case and the --mount case we can pull it from the image itself and put it in the measured archive region. But when the app mounts a file system, the myst command cannot know about it ahead of time (it has to be known when the image is created).